frontend/server.js

```
@@ -119,10 +112,22 @@ async function main() {
             allowHttpForRedirectUrl: dev,
             isB2C: true,
           },
-          (profile, done) => {
-            done(null, {
-              id: profile.sub,
+          async (profile, done) => {
+            const response = await getOrCreateUser({
```

If this fails, can we propagate the error up to passport?

---

Added `try/catch` to bubble up errors encountered by either the local or OIDC (Open ID Connect) Passport Strategies.